src/com/tw/go/plugin/cmd/Console.java

```diff
@@ -13,8 +13,12 @@ public static CommandLine createCommand(String... args) {
         gitCmd.addArguments(args);
         return gitCmd;
     }
-
+    
     public static ConsoleResult runOrBomb(CommandLine commandLine, File workingDir, ProcessOutputStreamConsumer stdOut, ProcessOutputStreamConsumer stdErr) {
+        return runOrBomb(commandLine, workingDir, stdOut, stdErr, commandLine.toString());
```

Comment:
if i understand this right, you are passing `commandLine.toString()` as `prettyMessage` and using it in `getMessage()`. But isn't that what's happening anyway?